node/typings.d.ts

```diff
@@ -100,6 +100,7 @@ interface Organization {
   paymentTerms: PaymentTerm[]
   status: string
   created: string
+  customFields: CustomField[]
```

Comment:
I think this should be an optional field `customFields?: CustomField[]`. Even if we set it every time an organization is saved it won't exist on old organizations, so the types should reflect that.

Reply:
✅Fixed